src/drivers/display/fb/lv_linux_fbdev.c

```diff
@@ -266,6 +272,34 @@ static void write_to_fb(lv_linux_fb_t * dsc, uint32_t fb_pos, const void * data,
 #endif
 }
 
+static void del_event_cb(lv_event_t * e)
+{
+    if (LV_EVENT_DELETE != lv_event_get_code(e))
+        return;
+
+    lv_display_t *disp = lv_event_get_target(e);
+    lv_linux_fb_t *dsc = lv_display_get_driver_data(disp);
+    if (!dsc) return;
+
+#if LV_LINUX_FBDEV_MMAP
+    if (dsc->fbp) {
```

Comment:
```suggestion
    if (dsc->fbp != MMAP_FAILED) {
```

If mmap fails, this will be MMAP_FAILED which maps to -1, not 0